msteams-platform/bots/how-to/teams-conversational-ai/ai-ux.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Custom engine agent user experience
+description: Learn about the user experience for custom engine agent
+ms.localizationpriority: medium
+ms.topic: overview
+ms.author: surbhigupta
+ms.date: 09/27/2024
+---
+
+# Custom engine agent user experience
+
+A custom engine agent transforms system interactions. For UX designers and developers, creating an exceptional user experience is crucial. This article details the steps, principles, and considerations for designing intuitive, user-centered interfaces that seamlessly integrate AI capabilities. The main goals are to simplify complex tasks, enhance productivity, and offer personalized experiences through adaptive learning. A custom engine agent includes features that enhance its functionality and integration within the Microsoft ecosystem:
```

Comment:
I don't think we need UX designers in here, custom engine target audience is developers.

Reply:
@aycabas Updated the input.